website/docs/docs/collaborate/explore-projects.md

```diff
@@ -217,6 +217,12 @@ Example of the details view for the model `supplies`:
 
 <Lightbox src="/img/docs/collaborate/dbt-explorer/example-model-details.png" width="100%" title="Example of resource details" />
 
+## Staging environment <Lifecycle status='beta' />
+
+dbt Explorer supports views for [Staging deployment enviornments](/docs/deploy/deploy-environments#staging-environment), in addition to the Production environment. This gives you a unique view into your pre-production data workflows with the same tools available in production. This allows you to control access to sensitive data and the added benefits of having a layer between development and production. Once the Staging environment is configured, it will be visible in the dbt Explorer landing page.
```

Comment:
```suggestion
dbt Explorer supports views for [Staging deployment enviornments](/docs/deploy/deploy-environments#staging-environment), in addition to the Production environment. This gives you a unique view into your pre-production data workflows with the same tools available in production. This allows you to control access to sensitive data and the added benefits of having a layer between development and production. Once the Staging environment is configured, it will be visible on the dbt Explorer landing page.
```